src/StripeTests/Entities/Discounts/DiscountTest.cs

```diff
@@ -6,6 +6,11 @@ namespace StripeTests
 
     public class DiscountTest : BaseStripeTest
     {
+        public DiscountTest(StripeMockFixture stripeMockFixture)
+            : base(stripeMockFixture)
+        {
+        }
+
```

Comment:
This test (and some others) doesn't rely on `GetFixture`, so it doesn't need to receive the `StripeMockFixture` instance. My codemod wasn't smart enough to figure that out though, and it's probably best to be consistent anyway.